lib/rubocop/cop/rspec/cop.rb

```diff
@@ -41,32 +33,28 @@ def relevant_file?(file)
         private
 
         def relevant_rubocop_rspec_file?(file)
-          rspec_pattern =~ file
+          self.class.rspec_pattern =~ file
```

Comment:
Use `match?`

Reply:
We're introducing `rubocop-performance` to tackle that https://github.com/rubocop-hq/rubocop-rspec/pull/970